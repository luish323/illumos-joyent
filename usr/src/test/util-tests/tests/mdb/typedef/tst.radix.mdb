::typedef -c lp64
::typedef 'struct foo { uint8_t foo[10]; }' foo_t
::typedef 'struct bar { uint8_t foo[0x10]; }' bar_t
::sizeof foo_t
::print -at foo_t
::sizeof bar_t
::print -at bar_t
