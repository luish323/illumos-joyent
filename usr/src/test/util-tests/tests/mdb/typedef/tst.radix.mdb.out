sizeof (foo_t) = 0xa
0 foo_t {
    0 uint8_t [10] foo 
}
sizeof (bar_t) = 0x10
0 bar_t {
    0 uint8_t [16] foo 
}
